lib/aruba/api.rb

```diff
@@ -1036,12 +1049,12 @@ def run_interactive(cmd)
     #   The input for the command
     def type(input)
       return close_input if "" == input
-      _write_interactive(_ensure_newline(input))
+      last_command.write(_ensure_newline(input))
     end
 
     # Close stdin
     def close_input
-      @interactive.stdin.close
+      last_command.close_io(:stdin)
```

Comment:
Nothing special about interactive commands anymore. See one of the comment above.